packages/syft/src/syft/custom_worker/runner_k8s.py

```diff
@@ -217,3 +284,23 @@ def _create_or_get(self, obj: APIObject) -> APIObject:
         else:
             obj.refresh()
         return obj
+
+    def _create_dockerconfig_json(
+        self,
+        reg_username: str,
+        reg_password: str,
+        reg_url: str,
+    ):
+        config = {
+            "auths": {
+                reg_url: {
+                    "username": reg_username,
+                    "password": reg_password,
+                    "auth": base64.b64encode(
+                        f"{reg_username}:{reg_password}".encode()
+                    ).decode(),
+                }
+            }
+        }
+        print(json.dumps(config, indent=4))
```

Comment:
MAYBE:  we could print this only in logger debug mode?

Reply:
Ah good catch